lib/src/create_report.dart

```diff
@@ -542,9 +536,14 @@ class _Issue {
 
   String markdown({@required String basePath}) {
     return [
+      '*',
+      '<details>',
```

Comment:
I think the lines below the `*` need to have indent. `package:markdown` will generate and empty `<ul><li></li></ul>`and a separate block for the rest.

Experimenting with `package:markdown` shows that having two extra spaces on the second line (e.g. `'  <details>'`) is enough, the rest of the lines will be put inside the `<li></li>`.